lib/tasks/add_navigation_links.rake

```diff
@@ -1,25 +1,69 @@
 # rubocop:disable Metrics/BlockLength
 namespace :navigation_links do
+  reading_icon = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
+  contact_icon = File.read(Rails.root.join("app/assets/images/twemoji/contact.svg")).freeze
+  thumb_up_icon = File.read(Rails.root.join("app/assets/images/twemoji/thumb-up.svg")).freeze
+  smart_icon = File.read(Rails.root.join("app/assets/images/twemoji/smart.svg")).freeze
+  look_icon = File.read(Rails.root.join("app/assets/images/twemoji/look.svg")).freeze
+  listing_icon = File.read(Rails.root.join("app/assets/images/twemoji/listing.svg")).freeze
+  mic_icon = File.read(Rails.root.join("app/assets/images/twemoji/mic.svg")).freeze
+  camera_icon = File.read(Rails.root.join("app/assets/images/twemoji/camera.svg")).freeze
+  tag_icon = File.read(Rails.root.join("app/assets/images/twemoji/tag.svg")).freeze
+  bulb_icon = File.read(Rails.root.join("app/assets/images/twemoji/bulb.svg")).freeze
+  shopping_icon = File.read(Rails.root.join("app/assets/images/twemoji/shopping.svg")).freeze
+  heart_icon = File.read(Rails.root.join("app/assets/images/twemoji/heart.svg")).freeze
+  rainbowdev = File.read(Rails.root.join("app/assets/images/rainbowdev.svg")).freeze
+
+  desc "Create navigation links for new forem"
+  task create: :environment do
+    NavigationLink.where(url: "/readinglist").first_or_create(
```

Comment:
Notice that my `where` statement has the relative URL, and not the absolute URL complete with `base_url` (like this: `NavigationLink.where(url: "#{base_url}/listings").first_or_create`).

However, my tests would only pass when I searched `url` this way.

In creating the actual link, however, I pass in an absolute link for the `url` attribute.

Reply:
I double-checked with Blazer, the URLs are indeed stored as relative URLs (except for the DEV specific shop link).

Reply:
Yes, the DEV Shop link is external, so it remains unchanged after applying the `strip_local_hostname` [callback](https://github.com/forem/forem/blob/ec949f9e04766c20db801f4d83660725c2e52ee9/app/models/navigation_link.rb#L29).

The default links for new forems are all local, so I think selecting by relative links is alright.